library/src/scala/IArray.scala

```diff
@@ -135,9 +131,6 @@ object IArray:
 
   /** Finds index of first occurrence of some value in this array after or at some start index. */
   extension [T](arr: IArray[T]) def indexOf(elem: T, from: Int = 0): Int =
-    // `asInstanceOf` needed because `elem` does not have type `arr.T`
-    // We could use `arr.iterator.indexOf(elem, from)` or `arr.indexWhere(_ == elem, from)`
-    // but these would incur some overhead.
     genericArrayOps(arr).indexOf(elem.asInstanceOf, from)
```

Comment:
The comment disappeared, but the `asInstanceOf` remains. Probably the comment should be preserved as well.

Reply:
Added back